src/nft/components/details/detailsV2/DataPageHeader.tsx

```diff
@@ -41,7 +41,7 @@ export const DataPageHeader = ({ asset }: { asset: GenieAsset }) => {
         </ThemedText.HeadlineMedium>
       </AssetText>
       <Row justifySelf="flex-end" width="min-content" gap="12px">
-        <BuyButton asset={asset} onDataPage />
+        <BuyButton asset={asset} dataPage />
```

Comment:
nit: `onDataPage` was a more clear var name imo

Reply:
I agree - unfortunately react auto assume every `on` variable is a function and does a lookup and it was throwing an error in the inspector

Reply:
`isDataPage` then? Just to make ti clear it's a bool

Reply:
`isDataPage` then? Just to make ti clear it's a bool